rs/ic_os/config/src/deployment_json.rs

```diff
@@ -320,6 +320,41 @@ mod test {
         }
     });
 
+    const DEPLOYMENT_STR_NO_LOGGING_HOSTS: &str = r#"{
+      "deployment": {
+        "name": "mainnet",
+        "mgmt_mac": null
+      },
+      "logging": {
+        "hosts": ""
+      },
+      "nns": {
+        "url": "https://wiki.internetcomputer.org/"
+      },
+      "resources": {
+        "memory": "490",
+        "cpu": "kvm"
+      }
+    }"#;
+
+    static DEPLOYMENT_STRUCT_NO_LOGGING_HOSTS: Lazy<DeploymentSettings> = Lazy::new(|| {
+        let hosts = [""].join(" ");
```

Comment:
nit: We should use `"".to_string()` here, or even `Logging { hosts: Default::default() }` below, instead.

Reply:
Done! 78386b089afb040f38a38d665d9461ea1c03dc96